dbt/contracts/graph/unparsed.py

```diff
@@ -85,3 +85,80 @@ class UnparsedMacro(APIObject):
 
 class UnparsedNode(APIObject):
     SCHEMA = UNPARSED_NODE_CONTRACT
+
+
+COLUMN_TEST_CONTRACT = {
+    'type': 'object',
+    'additionalProperties': False,
+    'properties': {
+        'name': {
+            'type': 'string',
+            'description': 'The name of the column this test is for',
+        },
+        'description': {
+            'type': 'string',
+            'description': 'The description of this test',
+        },
+        'tests': {
+            'type': 'array',
+            'items': {
+                "anyOf": [
+                    # 'not_null', 'unique', ...
+                    {'type': 'string'},
+                    # 'relationships: {...}', 'accepted_values: {...}'
+                    {'type': 'object', 'additionalProperties': True}
+                ],
+            },
+            'description': 'The list of tests to perform',
+        },
+    },
+    'required': ['name'],
+}
+
+
+UNPARSED_NODE_UPDATE_CONTRACT = {
+    'type': 'object',
+    'additionalProperties': False,
+    'description': (
+        'A collection of the unparsed node updates, as provided in the '
+        '"models" section of schema.yml'
+    ),
+    'properties': {
+        'name': {
+            'type': 'string',
+            'description': (
+                'The name of this node, which is the name of the model it'
+                'refers to'
+            ),
+            'minLength': 1,
+            'maxLength': 127,
```

Comment:
are we going to have the same issue with really long auto-generated test names here?

Reply:
I don't think so? I'm pretty sure this is the un-namespaced name of a model that's actually in the database (so not ephemeral), which means for db reasons it has to be a maximum of 127 characters, right?

Reply:
I think it's prudent to get rid of the length restriction anyway. This field on unparsed node had this restriction removed in be8b9c0b0bf4815fe5855bea97b6b250c48ab4b8, for consistency let's remove it here.